doc/build_html

```diff
@@ -5,4 +5,10 @@ zip -9 doc/source/_static/examples.zip examples/*
 tar -czf doc/source/_static/examples.tgz examples
 popd
 
-sphinx-build -M html "." "../build/html" 
+if command -v uv &> /dev/null; then
```

Comment:
This change could be made a lot easier in ci.yml line 74 without any side effects.

Reply:
(1) No, since the call to `sphinx-build` is in `build_html`.  It would fail without `uv run` (or activating the venv)
(2) By doing it this way it can work either with or without `uv`.  

I'm not ready to propose switching over the entire project yet.  (e.g. `python -m build` -> `uv build` and some `pyproject.toml` changes)

Reply:
if you out uv in front of build_html it should activate the venv which will then be handed to sphinx.

Reply:
Yeah I expected you to try before writing, however if I run "workon" before calling build_html it uses the environment assigned, so I do not understand what the problem is with running "uv" before the build_html command. venv is normally inherited across shell sessions.

your python_env function have one floor, I have at the moment 8 different venv, and when I login to my vm, I start by "workon <current project>".

Reply:
Interesting, this is my first hearing of `workon`.

For the CI, `uv sync` _creates_ a venv, but does not activate it.  So I had to activate it.

I think this is resolved to your satisfaction?  `build_html` is untouched compared to `dev`.